src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -51,10 +62,17 @@ export const Spotlight: FC = () => {
       case 'Enter':
         const value = e.currentTarget.value
         dispatch({ type: 'SEND_REQUEST', request: value })
-        setTimeout(
-          () => dispatch({ type: 'RECEIVE_RESPONSE', response: `I heard "${value}".` }),
-          500,
-        )
+        serviceStub.createRequestResponseOperation('nlp', 'getNlpIntent', value)
```

Comment:
do you have any patterns in the project to separate concerns of component VS server-side communication (like redux middleware or something similar)?

Reply:
this is v much poc at this stage

Reply:
we'll pbly want to use redux-observable at some point anyway, to handle concurrent requests.